src/components/Navibar.js

```diff
@@ -23,24 +24,27 @@ export default class Navibar extends Component {
         });
     }
     render() {
+        const logo = {
+            src: "/images/Agroeasylogo.png"
+        }
```

Comment:
rename `logo` to `LOGO` it to the constants page

Reply:
Done